src/index.html

```diff
@@ -1,13 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27&Prime; Retina 5K Monoblock (MNED2UA/A)"
+        class="card__img"
+      >
+      <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__rating">
+        <div class="card__stars stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+      <div class="card__price">
```

Comment:
See checklist about empty lines.
```suggestion
      </div>
      
      <div class="card__price">
```